velox/docs/functions/presto/conversion.rst

```diff
@@ -495,6 +520,64 @@ Valid examples
   SELECT cast(date '1970-01-01' as timestamp); -- 1970-01-01 00:00:00
   SELECT cast(date '2012-03-09' as timestamp); -- 2012-03-09 00:00:00
 
+From timestamp with time zone
```

Comment:
TIMESTAMP WITH TIME ZONE

Reply:
I followed the format convention in the same doc. Are we sure we want to make exception for this new entry?

Reply:
We want to fix the doc, but it would be good to not add more text like the one that needs fixing.